source/Coop/Mod/Managers/ClientCharacterCreatorManager.cs

```diff
@@ -78,16 +82,22 @@ public override void OnLoadFinished()
         private void SkipCharacterCreation()
         {
             CharacterCreationState characterCreationState = GameStateManager.Current.ActiveState as CharacterCreationState;
-            bool flag = CharacterObject.PlayerCharacter.Culture == null;
+            bool flag = characterCreationState.CurrentStage is CharacterCreationCultureStage;
             if (flag)
             {
-                CultureObject culture = CharacterCreationContentBase.Instance.GetCultures().FirstOrDefault<CultureObject>();
+                CultureObject culture = CharacterCreationContentBase.Instance.GetCultures().GetRandomElementInefficiently();
                 CharacterCreationContentBase.Instance.SetSelectedCulture(culture, characterCreationState.CharacterCreation);
                 characterCreationState.NextStage();
             }
             bool flag2 = characterCreationState.CurrentStage is CharacterCreationFaceGeneratorStage;
             if (flag2)
             {
+                CharacterCreationFaceGeneratorView stageView = characterCreationState.CurrentStage.Listener as CharacterCreationFaceGeneratorView;
+                BodyGeneratorView bgv = (BodyGeneratorView)stageView.GetType().GetField("_faceGeneratorView", BindingFlags.NonPublic | BindingFlags.Instance).GetValue(stageView);
+
+                FaceGenVM facegen = bgv.DataSource;
+
+                facegen.FaceProperties.Randomize();

```

Comment:
Is this intended to always randomize the face during character generation? Should there be a todo to handle the clients actually creating a character?

Reply:
This is to skip character creation is only for debugging purposes and will be omitted in the released version. Character creation already works.